cmd/root.go

```diff
@@ -14,11 +16,19 @@ var (
 	tfstate bool
 	file    []byte
 	path    string
+	debug   bool
+	logsOut io.Writer = ioutil.Discard
 
 	rootCmd = &cobra.Command{
 		Use:   "inframap",
 		Short: "Reads the TFState or HCL to generate a Graphical view",
 		Long:  "Reads the TFState or HCL to generate a Graphical view with Nodes and Edges.",
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if debug {
+				logsOut = os.Stdout
+			}
+			log.Init(logsOut, debug)
+		},
```

Comment:
I would prefer not to include TC for this, copy the part you  need (activation of TF logs) and open an issue to add Debugging to IM (if we do not have one already)